sunbeam/commands/bootstrap.py

```diff
@@ -62,20 +67,27 @@
 @click.command()
 @click.option(
     "--role",
-    default="converged",
-    type=click.Choice(["control", "compute", "converged"], case_sensitive=False),
+    multiple=True,
+    default=["control", "compute"],
+    type=click.Choice(["control", "compute", "storage"], case_sensitive=False),
     help="Specify whether the node will be a control node, a "
-    "compute node, or a converged node (default)",
+    "compute node or a storage node. Defaults to all the roles.",
 )
 def bootstrap(role: str) -> None:
     """Bootstrap the local node.
 
     Initialize the sunbeam cluster.
     """
-    node_role = Role[role.upper()]
+    node_roles = [Role[role_.upper()] for role_ in role]
+
+    is_control_node = any([role_.is_control_node() for role_ in node_roles])
+    is_compute_node = any([role_.is_compute_node() for role_ in node_roles])
+    is_storage_node = any([role_.is_storage_node() for role_ in node_roles])
```

Comment:
Little nit, collecting into a list is unnecessary
```suggestion
    is_control_node = any(role_.is_control_node() for role_ in node_roles)
    is_compute_node = any(role_.is_compute_node() for role_ in node_roles)
    is_storage_node = any(role_.is_storage_node() for role_ in node_roles)
```

Reply:
Done